### PR TITLE
Add monitools container

### PR DESCRIPTION
**Pull request by adrianriobo**
The idea is have a container holding the cli binary. The container then will copy the binary to the host, run it and pick the results

### CODE REVIEW

File: images/build/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Build the manager binary
+FROM registry.access.redhat.com/ubi8/go-toolset:1.14.12 as builder
+
+USER root
+WORKDIR /workspace
+COPY . .
+RUN make compile
```
Comment (praveenkumar) [inline images/build/Dockerfile:7]: Better to use `make build` to have platform specific binary instead of cross binary.

Comment (adrianriobo) [reply images/build/Dockerfile]: changed

File: images/build/entrypoint.sh [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+    exit 1
+fi
+
+if [[ -z "${TARGET_HOST_KEY_PATH}" ]]; then
+    printf "Please set up TARGET_HOST_KEY_PATH Env\n"
+    exit 1
+fi
+
+# Copy monictl to target host
+scp -o "StrictHostKeyChecking=accept-new" \
```
Comment (praveenkumar) [inline images/build/entrypoint.sh:24]: I think we should use `StrictHostKeyChecking=no` and also `-o UserKnownHostsFile=/dev/null`

Comment (adrianriobo) [reply images/build/entrypoint.sh:24]: Seems ok to me as it is one shot run, so not need to keep entries

**Review by jsliacan — APPROVED** [review]
I don't have any significant suggestions, LGTM.

**Pull request merged**